test/parallel_api/ranges/std_ranges_adjacent_find.pass.cpp

```diff
@@ -0,0 +1,32 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "std_ranges_test.h"
+
+std::int32_t
+main()
+{
+#if _ENABLE_STD_RANGES_TESTING
+    using namespace test_std_ranges;
+    namespace dpl_ranges = oneapi::dpl::ranges;
+
+    test_range_algo<0>{}(dpl_ranges::adjacent_find,  std::ranges::adjacent_find, binary_pred);
```

Comment:
Minor thing: extra space. `dpl_ranges::adjacent_find,  std::ranges::adjacent_find` -> `dpl_ranges::adjacent_find, std::ranges::adjacent_find`.
It also is in other tests.